homeworks/src/homework6/ArrayDataException.java

```diff
@@ -0,0 +1,15 @@
+package homework6;
+
+public class ArrayDataException extends Exception {
+
+    private static String errorMessage;
```

Comment:
You did a bridge between multiple exceptions. If you will create **arrayDataException1**=ArrayDataException("**message1**") and call **arrayDataException1**.getErrorMessage()->"**message1**". If will create **arrayDataException2**=ArrayDataException("**message2**"). And after that call **arrayDataException1**.getErrorMessage()->**message2.** It's may create serious issues